integration/jnifuse/jnifuse-java/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -34,11 +34,13 @@
  */
 public abstract class AbstractFuseFileSystem implements FuseFileSystem {
 
+  private static final Logger LOG = LoggerFactory.getLogger(
```

Comment:
unneccessary line wrap. Are you using line width of 80? Alluxio code convention is 100

Reply:
Done